src/guide/buy_btc.md

```diff
@@ -7,10 +7,13 @@ This section of the Breez SDK documentation provides an example on purchasing Bi
 <div slot="title">Rust</div>
 <section>
 
-```rust
-// TODO add docs
+```rust,ignore
+let res = sdk.buy_bitcoin(
+    BuyBitcoinRequest {
+        provider: BuyBitcoinProvider::Moonpay,
+        opening_fee_params: None})
```

Comment:
Can we not remove the `opening_fee_params` since it's null by default?

Reply:
Unfortunately that doesn't work in Rust, it's only for the uniffi languages.